docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Running an Accuracy Classification model in Python with Flojoy
+title: Running an Accuracy Classification Model in Python with Flojoy
 description: The ACCURACY node takes two dataframes with the true and predicted labels from a classification task, and indicates whether the prediction was correct or not. These dataframes should both be single columns.
```

Comment:
"...and indicates what proportion of the predictions were correct." (no "or not" at the end).